notebooks/110-ct-segmentation-quantize/110-ct-segmentation-quantize-nncf.ipynb

```diff
@@ -0,0 +1,795 @@
+    "- Convert and Quantize a Segmentation Model and Show Live Inference (this notebook)\n",
+    "- [Live Inference and Benchmark CT-scan data](../210-ct-scan-live-inference/210-ct-scan-live-inference.ipynb) \n",
+    "\n",
+    "## Instructions\n",
+    "\n",
+    "This notebook needs a trained UNet model. We provide a pretrained model trained for 20 epochs with the full [Kits-19](https://github.com/neheller/kits19) frames dataset, which has an F1 score on the validation set of 0.9. The training code is available in [this notebook](pytorch-monai-training.ipynb). \n",
+    "\n",
+    "NNCF for PyTorch models requires a C++ compiler. On Windows, please install Microsoft Visual Studio Build Tools, and start Jupyter from a _Developer Command Prompt_.\n",
+    "\n",
+    "Running this notebook with the full dataset will take a long time. For demonstration purposes, this tutorial will download one converted CT scan and use that scan for quantization and inference."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "import os\n",
+    "import random\n",
+    "import sys\n",
+    "import time\n",
+    "import warnings\n",
+    "import zipfile\n",
+    "from pathlib import Path\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\", category=UserWarning)\n",
+    "\n",
+    "import cv2\n",
+    "import matplotlib.pyplot as plt\n",
+    "import monai\n",
+    "import numpy as np\n",
+    "import torch\n",
+    "from monai.transforms import LoadImage\n",
+    "from nncf.common.utils.logger import set_log_level\n",
+    "from openvino.inference_engine import IECore\n",
+    "from torch.jit import TracerWarning\n",
+    "from torchmetrics import F1\n",
+    "\n",
+    "set_log_level(logging.ERROR)  # Disables all NNCF info and warning messages\n",
+    "\n",
+    "sys.path.append(\"../utils\")\n",
+    "from models.custom_segmentation import SegmentationModel\n",
+    "from notebook_utils import NotebookAlert, benchmark_model, download_file, show_live_inference\n",
+    "\n",
+    "try:\n",
+    "    import subprocess\n",
+    "\n",
+    "    from nncf import NNCFConfig\n",
+    "    from nncf.torch import create_compressed_model, register_default_init_args\n",
+    "except subprocess.CalledProcessError:\n",
+    "    message = \"WARNING: Please install a C++ compiler to use this notebook.\"\n",
+    "    if sys.platform == \"win32\":\n",
+    "        message += (\n",
+    "            \" If you installed Visual Studio Build Tools, please open a Developer \"\n",
+    "            \"Command Prompt and start Jupyter from there.\"\n",
+    "        )\n",
+    "    NotebookAlert(message=message, alert_class=\"warning\")\n",
+    "    raise"
+   ]
```

Comment:
Windows 11. Microsoft Visual Studio 2022 Community. Running in Developer Command prompt. cl.exe path added into environmental variables. Python version 3.9.13. I got this error.

![image](https://user-images.githubusercontent.com/10940214/174420703-5e7b1300-0cc0-4f38-b00a-7b037916880e.png)

Reply:
It seems tricky to get this right on Windows. I think the developer command prompt works well if you use the x64 prompt and the version of Python that is installed with Visual Studio, but that's not something that most Python developers will be familiar with. I added the pathfinder cell back to the top, and removed the recommendation to use a developer prompt. 

I added a link to VS install instructions to the notebook (and also mentioned what to install for macOS and Linux). OpenVINO system requirements specify Visual Studio 2019, and that's what we test, so I added a link to that. It may work well with VS 2022 though - but I cannot test that at the moment.

The error message in the screenshot shows that the default installation root for 2022 is different. I would try this: start Jupyter from a regular, non developer prompt, and change the VS_INSTALL_DIR line in the pathfinder cell to:

`VS_INSTALL_DIR = r"C:/Program Files/Microsoft Visual Studio"`